pyproject.toml

```diff
@@ -82,57 +80,49 @@ warn_unused_ignores = true
 warn_redundant_casts = true
 warn_unused_configs = true
 exclude = ["scripts/"]
-plugins = [
-  "pydantic.mypy",
-]
+plugins = ["pydantic.mypy"]
 
 [[tool.mypy.overrides]]
 module = "sdx.schema.fhirx"
-disable_error_code = [
-  "name-defined",
-  "valid-type",
-]
+disable_error_code = ["name-defined", "valid-type"]
 
 [tool.pytest.ini_options]
-testpaths = [
-    "tests",
-]
+testpaths = ["tests"]
 
 [tool.ruff]
 line-length = 79
 force-exclude = true
 src = ["./"]
-exclude = [
-  'docs',
-]
+exclude = ['docs']
 
 [tool.ruff.lint]
-select = [
-  "E",   # pycodestyle
-  "F",   # pyflakes
-  "D",   # pydocstyle
-  "YTT", # flake8-2020
-  "RUF", # Ruff-specific rules
-  "I001",  # isort
-]
+select = ["E", "F", "D", "YTT", "RUF", "I001"]
 fixable = ["I001", "F401"]
 
 [tool.ruff.lint.pydocstyle]
 convention = "numpy"
 
 [tool.ruff.lint.isort]
-# Use a single line between direct and from import
 lines-between-types = 1
 
 [tool.ruff.format]
 quote-style = "single"
 
 [tool.vulture]
 exclude = ["tests"]
-ignore_decorators = []
-ignore_names = []
-make_whitelist = true
 min_confidence = 80
 paths = ["./"]
 sort_by_size = true
-verbose = false
+
+[project.optional-dependencies]
+dicom = ["pydicom >=2.5"]
+dev = ["pytest", "ruff", "pre-commit"]
+
+[tool.poetry.group.dicom.dependencies]
+# This section defines the 'dicom' dependency group used in CI via:
+#   poetry install --with dicom
+# The same package is also defined under [project.optional-dependencies]
+# to allow user installs via:
+#   poetry install -E dicom
+# Keep both entries synchronized manually.
+pydicom = ">=2.4.0"
```

Comment:
you probably shouldn't specify the name and version of the package again. just need to define it as optional; ex: https://python-poetry.org/docs/managing-dependencies/#optional-groups

also changes the name from `[tool.poetry.group.dicom.dependencies]` to `[tool.poetry.group.dicom]`

None of these comments are necessary. please avoid using unnecessary comments